validator/src/main/java/com/amazon/aoc/callers/HttpCaller.java

```diff
@@ -29,7 +30,7 @@ public SampleAppResponse callSampleApp() throws Exception {
 
     AtomicReference<SampleAppResponse> sampleAppResponseAtomicReference = new AtomicReference<>();
     RetryHelper.retry(
-        60,
+        3,
```

Comment:
we will still need a bigger retry count, service launching time in different platform varies. 3 times might not be enough in some cases.

Reply:
Yes. I agree. Maybe we can probably keep this to 10? I feel 60 times retry is too much especially we will be using this module on Github actions where we don't have any control to terminate it we just have to wait for 60 retries to complete.

Reply:
i'm not sure if 10 would work, because it normally takes 10+ times to get it work for ecs test. probably 30?

Reply:
Okay we can do 30 then.